src/main/scala/org/broadinstitute/dsde/agora/server/AgoraConfig.scala

```diff
@@ -90,4 +92,26 @@ object AgoraConfig {
       case _ => baseUrl
     }
   }
+
+  implicit class EnhancedScalaConfig(val config: Config) extends AnyVal {
```

Comment:
FWIW we moved away from this (we had more or less the same code on our side) and have since switched to [ficus](https://github.com/iheartradio/ficus). The ultimate goal was to not roll our own when there are real libraries to solve this issue, don't remember why we settled on ficus